### ISSUE TITLE
OCprofile fan speed setting

### ISSUE THREAD

**Issue by khayto** [RainbowMiner/RainbowMiner#974]
hey, idk if you forgot so im just reposting as a reminder. when ever you got time, would like to have a fanspeed field in OCprofile configs. -setFanSpeed is the command in a batch file i believe.

**Comment by RainbowMiner** [comment]
Sure, thank you for the reminder.

**Comment by samdave123** [comment]
What would be really ideal is to have a target temp and have the fanspeed dynamically adjust to meet that target temp. I don't think it can be done with MSI Afterburner or Nvidia Inspector, but last time I used Nicehack they had auto fan control which seemed to target a set temperature.

**Comment by bing281** [comment]
would really like this also as it would complete the OCprofiles.   Suggestion set it up as follows temp1=40, fan1=50, temp2=50, fan2=60.... and so on.   

That way you could make a fan graph basically.

**Comment by nevermind19** [comment]
Yes it's really needed for my devices !Cause unfortunatly i have the gpus very hot (more than 80°C) and the autocontrol fan is not increase...please add this feature

Thanks

**Comment by Nc2meu2** [comment]
Hello, has this been implemented?
Can someone give me step-by-step to set a max temperature for my gpus?

**Comment by RainbowMiner** [comment]
Sorry, I still haven't had the time to implement fan curves. I'll move it up on the prio list.

**Comment by Nc2meu2** [comment]
That would be a plus, especially now that the days are getting hotter, my gpus will need me to schedule them to take a break or turn them off.

**Comment by RainbowMiner** [comment]
Duplicate of #849 (same request, reposted as a reminder). Keeping #849 as the single thread for this. Fan speed is currently only read for monitoring, not settable in OC profiles, so the request is still valid - it is just tracked in one place now.
